src/components/Form/Form.tsx

```diff
@@ -48,13 +50,23 @@ const Form = (props: FormProps) => {
 
 	useEffect(() => {
 		setTopComponentHeight(topComponentRef.current?.offsetHeight);
-		setTopComponentDrawerHeight && setTopComponentDrawerHeight(`${topComponentRef.current?.offsetHeight}px`);
 
-		if (formContentRef?.current?.children[0] && setFormLayoutPadding) {
-			const layoutPaddingTop = window?.getComputedStyle(formContentRef.current.children[0]).getPropertyValue("padding-top");
-			const layoutPaddingBottom = window?.getComputedStyle(formContentRef.current.children[0]).getPropertyValue("padding-bottom");
-			setFormLayoutPadding(`${layoutPaddingTop} - ${layoutPaddingBottom}`);
-		}
+		dispatchRef && topComponentRef.current && dispatchRef({
+			type: "update",
+			ref: {
+				id: "topComponentDrawerRef",

```

Comment:
Same comment as RefsReducer:
```suggestion
				topComponentDrawerRef: topComponentRef.current
```